addons/editor-stepping/addon.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Highlight currently executing blocks",
+  "name": "Currently executing block highlight",
```

Comment:
This name is also kinda weird.

Reply:
Maybe "Highlight around currently executing block"?

Reply:
@mxmou Yes

Reply:
I think this addon needs a shorter name - @Hans5958 suggested "Editor Stepping" in a comment below. I'll use "Running block border" for now.